core/db.py

```
@@ -294,3 +322,119 @@ async def init_db_async() -> None:
+
+
+def get_db_connection():
+    """Get a database connection for testing purposes.
+
+    RU: Возвращает соединение c базой данных для тестирования.
+    EN: Returns a database connection for testing purposes.
+    """
+    try:
+        return _RAW_ENGINE.connect()
+    except Exception as exc:
+        logger.exception("Failed to get database connection", exc_info=exc)
```

Flake8 (7.3.0)

[error] undefined name 'logger'

(F821)